cmd_config.go

```
@@ -0,0 +1,240 @@
+	}
+
+	// Reset config command
+	resetCmd := &cobra.Command{
+		Use:   "reset",
+		Short: "Reset configuration to defaults",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			GlobalLogger.Warning.Printf("This will reset configuration to defaults: %s\n", configPath)
+			fmt.Print("Are you sure? (y/N): ")
+			
+			var response string
+			fmt.Scanln(&response)
```

golangci-lint (1.64.8)

Error return value of `fmt.Scanln` is not checked

(errcheck)